spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -93,6 +94,12 @@ def get_parser():
         help='Copy the header of the source image (specified in -i) to the destination image (specified here) '
              'and save it into a new image (specified in -o)',
         required = False)
+    header.add_argument(
+        '-set-sform-to-qform',
+        help='Set the sform matrix of all input images to their respective qform matrix. '
+             'This operation is applied anytime an image is read in by `sct_image`.',
```

Comment:
This description says "of all input images", but is that accurate? 

I notice we only apply `sct_image -set-sform-to-qform` to only the first image, `fname_in[0]`:

https://github.com/neuropoly/spinalcordtoolbox/blob/ff54a929cfe447d40958364ee1a35c77b04189e4/spinalcordtoolbox/scripts/sct_image.py#L318-L321

Reply:
Ah, I see now. Only `-concat` uses multiple images. And, in that case, setting sform to qform doesn't make sense?

I will rewrite some of the help descriptions to make single/multi image behavior more clear.

Reply:
Ah, I see now. Most functions use only the first image, but some use a list of images. 

This is... weird, and not communicated well to the user. I think that work could be done here, but it's outside the scope of this PR, so I'll just make some notes about it.

Reply:
Done in db03173.

Reply:
> This is... weird, and not communicated well to the user. I think that work could be done here, but it might be outside the scope of this PR, so I'll just make some notes about it.

100% agreed! i'm glad you caught (and fixed) it!